src/util/misc/projectStroke.ts

```diff
@@ -1,12 +1,20 @@
 import { Point } from '../../point.class';
-import { createVector, getBisector } from './vectors';
-import { StrokeLineJoin } from '../../typedefs';
+import { calcAngleBetweenVectors, createVector, getBisector, getOrthogonalUnitVector } from './vectors';
+import { StrokeLineJoin, TDegree } from '../../typedefs';
+import { degreesToRadians } from './radiansDegreesConversion';
+import { hypot } from './hypot'
+
+const PiBy2 = Math.PI / 2
+
 /**
- * Project stroke width on points returning 2 projections for each point as follows:
- * - `miter`: 2 points corresponding to the outer boundary and the inner boundary of stroke.
- * - `bevel`: 2 points corresponding to the bevel boundaries, tangent to the bisector.
- * - `round`: same as `bevel`
+ * Project stroke width on points returning projections for each point as follows:
+ * - `miter`: 1 point corresponding to the outer boundary and the inner boundary of stroke.
+ * - `bevel`: 2 points corresponding to the bevel possible boundaries, orthogonal to the stroke.
+ * - `round`: same as `bevel` when it has no skew, with skew are 4 points.
  * Used to calculate object's bounding box
+ * 
+ * @see https://github.com/fabricjs/fabric.js/pull/8083
```

Comment:
Yes, when I migrate the documentation to this PR, I will create anchors for each case, and change the link in the code. That way, it will be easier to find the documentation.